package.json

```diff
@@ -34,10 +34,10 @@
   },
   "homepage": "https://github.com/kefirjs/kefir-test-utils#readme",
   "dependencies": {
+    "@sinonjs/fake-timers": "^7.1.2",
     "@types/kefir": "^3.8.0",
-    "@types/lolex": "^2.0.0",
     "@types/node": "^14.0.1",
-    "lolex": "^2.0.0"
+    "@types/sinonjs__fake-timers": "^6.0.3"
```

Comment:
Does the latest version of `fake-timers` come w/ types? I think it might.

Reply:
I'll check the release notes, I don't use TS myself.

Reply:
I'll be upgrading fake timers to latest version, there have been quite a few releases since I first opened this PR.

In a recent release (8.0.0) types have been [removed](https://github.com/sinonjs/fake-timers/blob/master/CHANGELOG.md#800--2021-09-15) from the library.